include/openssl/x509_vfy.h.in

```diff
@@ -675,9 +675,15 @@ void X509_STORE_CTX_set0_rpk(X509_STORE_CTX *ctx, EVP_PKEY *target);
 void X509_STORE_CTX_set0_verified_chain(X509_STORE_CTX *c, STACK_OF(X509) *sk);
 void X509_STORE_CTX_set0_crls(X509_STORE_CTX *ctx, STACK_OF(X509_CRL) *sk);
 int X509_STORE_CTX_set_purpose(X509_STORE_CTX *ctx, int purpose);
+#define X509_STORE_CTX_set_purpose_ex(ctx, purpose, override) \
+                X509_STORE_CTX_purpose_inherit_ex(ctx, 0, purpose, 0, override)
 int X509_STORE_CTX_set_trust(X509_STORE_CTX *ctx, int trust);
+#define X509_STORE_CTX_set_trust_ex(ctx, trust, override) \
+                X509_STORE_CTX_purpose_inherit_ex(ctx, 0, 0, trust, 0, override)
```

Comment:
The 'hanging' indentation is not right - it should be just 4 chars.
You can check the coding style using `util/check-format.pl`.